core/src/main/scala/kafka/log/UnifiedLog.scala

```diff
@@ -111,12 +110,7 @@ class UnifiedLog(@volatile var logStartOffset: Long,
 
   import kafka.log.UnifiedLog._
 
-  private val metricsGroup = new KafkaMetricsGroup(this.getClass) {
-    // For compatibility, metrics are defined to be under `Log` class
```

Comment:
Could you please keep those comments?

Reply:
Already add them back and I modified the comments in `LogSegment.java` a little bit to fit the current change since there's no more overriding.